### DISCUSSION TITLE
Emulator crashes.

### DISCUSSION THREAD

**Discussion by therealfuranku** [skyline-emu/skyline#881] [Q&A]
So I decided to play Celeste and the game just crashed and back to menu. My phone has Vulkan Support but idk why it keeps crashing and worstly I can't play the game.

my phone specs (low end device, not intended to play 3d intensive games)

Device Name: Oppo A16 (CPH2269)
OS: Android 11, ColorOS 11.1
Chipset: MediaTek MT6765G Helio G35 (12 nm)
CPU: Octa-core (4x2.3 GHz Cortex-A53 & 4x1.8 GHz Cortex-A53)
GPU: PowerVR GE8320

**Comment by iniMerchant64** [comment]
Itll never work. PowerVR gpus are a meme for the most part. None of them would be able to play skyline
